### PR TITLE
fix: ingredient doubles continues

### PR DESCRIPTION
**Pull request by aleene**
### Description
More doubles in the ingredient and related taxonomies removed

### CODE REVIEW

**Comment by aleene** [comment]
@stephanegigandet the errors that are left seem to be a result of the taxonomies or tests themselves.

File: taxonomies/additives.txt [hunk 1]
```diff
@@ -1919,7 +1919,7 @@ cs:E160, E160 food additive
 da:E160, Carotenoider, Karotenoid, Carotenoid, Blandede carotener
 de:E160, Carotinoide
 el:E160, E160 food additive
-es:E160, Carotenoides, Caroteno
+es:E160, Carotenoidesmonoestearato-de-glicerina
```
Comment (stephanegigandet) [inline taxonomies/additives.txt:1922]: This one looks strange.

Comment (stephanegigandet) [reply taxonomies/additives.txt]: I fixed it.

**Review by stephanegigandet — APPROVED** [review]
Thank you very much for the huge amount of work @aleene !

**Pull request merged**